.github/global.yml

```
@@ -124,6 +124,8 @@
   description: issues about tests or the test infrastructure
   color: "fff2cc"
   aliases: [type-new_test]
+- name: type::deprecation
+  description: requests removal of deprecated feature(s)
```

Let's use the color defined for other types for consistency

```suggestion
  description: requests removal of deprecated feature(s)
  color: "fff2cc"
```